src/Api/Controllers/CiphersController.cs

```diff
@@ -179,6 +179,7 @@ public async Task<CipherResponseModel> Put(string id, [FromBody] CipherRequestMo
                 throw new NotFoundException();
             }
 
+            var collectionIds = (await _collectionCipherRepository.GetManyByUserIdCipherIdAsync(userId, new Guid(id))).Select(c => c.CollectionId).ToList();
```

Comment:
While in here can we change the method parameter to just be a `Guid id` instead of `string id` and replace both instances of `new Guid(id)`.

Reply:
@justindbaur Updated!